filebeat/module/kafka/log/_meta/fields.yml

```diff
@@ -30,10 +31,12 @@
           description: >
             Java class the trace is coming from.
         - name: message
+          # Should this be mapped to `message` instead`? How is it different from the above message?
```

Comment:
This is for the optional field with a full stack trace.

I'd rather move this to the `keyword` first indexing convention for consistency, and add `text` right away as a multi-field.

Reply:
I think if we call the field `message` it should be `text` by default even if it's somewhere hidden under a prefix.

This could also hit `ignore_above` so we rather keep it as text. Not really good reasons to aggregate on these fields I think.

Reply:
Keeping it for now as is.